config/routes.rb

```diff
@@ -1,5 +1,5 @@
 MANIACChess::Application.routes.draw do
-  devise_for :users
+  devise_for :users, :controllers => { :omniauth_callbacks => "users/omniauth_callbacks" }
```

Comment:
Ahh, I understand now. I'm still not used to the terminology. 

I was looking at multiple resources and probably added  `=> "users/` by mistake. Thanks for catching that!